src/osp/Active/SysRender.h

```diff
@@ -113,6 +113,11 @@ struct RenderGroup
 
 struct ACtxRenderGroups
 {
+    // Required for std::is_copy_assignable to work properly inside of entt::any
+    ACtxRenderGroups() = default;
+    ACtxRenderGroups(ACtxRenderGroups const& copy) = delete;
```

Comment:
Are you meaning that you're intentionally disabling copying?

Reply:
If so, probably should say as much.

But why bother disabling copying?

Reply:
Some members of this struct are not copyable, making the struct implicitly uncopyable. I'm unsure of the exact details, but not writing this explicitly this causes a compile error.